### ISSUE TITLE
Unable to switch between wallets on Android app (2.4.2)

### ISSUE THREAD

**Issue by thephez** [bitpay/copay#4687]
On version 2.4.2 of the Android wallet (Samsung Galaxy S4 - KitKat), I am unable to switch between multiple wallets.  I've tried updating twice and had to revert both times.  The sidebar menu showing my list of wallets opens properly, but clicking on a wallet entry in the list just closes the menu without switching to that wallet.  The Settings option on that same menu works properly.  It acts as if the wallet menu items are either not initialized or are all pointing to the same wallet.  I did not notice any glaring issues in the Session Log.

I saw something tagged 2.5.0 a couple days ago, but don't know if it's been released yet.  If so, I will update to it when I have a chance and see if the issue persists.

**Comment by matiu** [comment]
Can you check the session log to see if there is any error shown there after you try to switch wallets?

**Comment by thephez** [comment]
I updated again (now to 2.5.0) and still see the same issue.  The only error I see is a `Subscription error: 525`  followed immediately by a `Subscription error: connection error:`.  I'm guessing that was a legitimate error from the wallet I've set up on my own BWS instance that is not currently accessible.

In the session log, I see `Route change from: "x" to "y"` messages for walletHome, add, buyandsell, amazon, about, logs, and preferencesGlobal, but never any messages regarding a switch from one wallet to another.
